qiskit/aer/aer_provider.ipynb

```diff
@@ -350,19 +349,16 @@
     }
    ],
    "source": [
-    "# Construct an empty quantum circuit\n",
+    "# Construct a quantum circuit with a given initial state\n",
```

Comment:
As above "Construct a quantum circuit that initialises qubits to a custom state"